### PR TITLE
[WIP] Start of a pull request checklist

### PR DESCRIPTION
**Pull request by elrayle**
Needs links to resources or more description.  Probably needs more steps.

[Viewable in UI](http://elr37-dev.library.cornell.edu/samvera_docs/pr-checklist.html)

### CODE REVIEW

**Comment by laritakr** [comment]
I'd like to see a section added regarding documentation. i.e. methods are documented appropriately, documentation is updated (or an issue is created  with detail to documented updates needed), etc.

**Comment by RudyOnRails** [comment]
Not sure if this is part of the implementation, but it should either be referenced, or entirely implemented (with fun md checkboxes) in the [PR template](https://github.com/samvera/hyrax/blob/master/.github/PULL_REQUEST_TEMPLATE.md):

File: pages/samvera/developer_community/pr_checklist.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - [ ] CLA:  The contributor has a valid CLA on file.
+  - [ ] Reviewer: At least one of the reviewers is from an institution other than the contributor's institution.
+- Code quality
+  - [ ] Code: Changes seem reasonable and meet the expected behavior stated in the related issue.  Changes follow best practices for the community.
+  - [ ] UI changes: Ad hoc testing of the expected behavior in the UI appears correct.  Expected behavior should be described in the related issue.
+  - [ ] UI existing: Ad hoc testing of related features in the UI appear to continue to operate correctly.
+  - [ ] Hound issues:  Hound issues are either fixed or comments identify why it doesn't need to be fixed.
+  - [ ] Test: There are tests written for all changes and new functionality.
+  - [ ] Test efficiency: Tests use factorybot `build` instead of `create` where ever possible.  This is especially important for repository objects that cause the object to be created in Fedora.
+  - [ ] Test coverage: Test coverage stayed the same or rose.  Review coveralls report for the PR.
```
Comment (no-reply) [inline pages/samvera/developer_community/pr_checklist.md]: could we replace this with something about adding unit tests that ensure correct behavior of new classes/modules/methods? and then I would add some guidance about feature tests (but that may be harder agree on).

**Comment by cjcolvar** [comment]
I just wanted to give a :+1: that this is looking good y'all.  Thanks for all your work on it.

**Pull request merged**